### ISSUE TITLE
Remove "Tweets mentioning this planning application" (PA-275)

### ISSUE THREAD

**Issue by oajirabot** [openaustralia/planningalerts#289]
<p>The tweets feature doesn't work if javascript is disabled. This is a problem for several reasons. The most important is that it stops that content from being accessible to people with screenreaders.</p>

<p>Also, if we were instead to capture the tweets on the server side we could do other things like send people email alerts when people mention their application on twitter.</p>

---

**Issue details imported from Jira:**

| Detail | Value |
| --- | --- |
| Assignee | Matthew Landauer |
| Reporter | Matthew Landauer |
| Created | 2011-02-10 19:03:23 |
| Updated | 2012-07-30 09:30:56 |

---
### Imported Jira Comments

**Matthew Landauer** - 2012-07-29 22:08:28

> <p>@Henare how would you feel about getting rid of this feature entirely?</p>
> 
> 
> <p>It's a bit troublesome that it's not accessible (i.e. doesn't work without javascript).</p>
> 
> 
> <p>Also, it's not been used very much and doesn't really appear to promote tweeting. So, is it the right thing to do? Is there something else that we could do that would be more effective?</p>

**Matthew Landauer** - 2012-07-29 22:12:06

> <p>I guess the more "up to date" way of doing something similar is to use the Twitter and/or Facebook tools for sharing.</p>

**Henare Degan** - 2012-07-30 09:14:14

> <p>Thanks Matthew - I'm happy for this to be removed on the basis that it hasn't been used much.</p>
> 
> 
> <p>It was also added pre-commenting as a way to promote conversation about DAs. Now that we have commenting, it's much better to focus on that.</p>
> 
> 
> <p>I'll add another ticket for an idea I have about that and I'm happy for this ticket to turn into removing this feature.</p>
>
